tests/profiles/googlefonts_test.py

```diff
@@ -3150,50 +3151,95 @@ def test_check_vertical_metrics_regressions(cabin_ttFonts):
                 api_gfonts_ttFont,
                 style,
                 remote_styles,
+                regular_ttFont,
+                regular_remote_style,
                 metadata_file,
                 family_metadata)
-    from copy import copy
+    from copy import deepcopy
+
+    remote = regular_ttFont([TTFont(f) for f in cabin_fonts])
+    ttFont = regular_ttFont([TTFont(f) for f in cabin_fonts])
+
+    # Cabin test family should match by default
+    assert_PASS(check(ttFont, remote),
+                'with a good family...')
+
+    # FAIL with a changed vertical metric value
+    remote2 = deepcopy(remote)
+    ttFont2 = deepcopy(ttFont)
+    ttFont2['OS/2'].sTypoAscender = 0
+    assert_results_contain(check(ttFont2, remote2),
+                           FAIL, 'bad-typo-ascender',
+                           'with a family which has an incorrect typoAscender...')
+
+    # TODO:
+    #   FAIL, "bad-typo-descender"
+    #   FAIL, "bad-hhea-ascender"
+    #   FAIL, "bad-hhea-descender"
```

Comment:
can you add code-tests for these as well?

Reply:
Sure. They didn't exist previously and I was quite lazy last week ;-)